lib/grape.rb

```diff
@@ -42,6 +42,10 @@ def self.deprecator
     @deprecator ||= ActiveSupport::Deprecation.new('2.0', 'Grape')
   end
 
+  def self.rack3?
+    Gem::Version.new(::Rack.release) >= Gem::Version.new('3')
```

Comment:
If you are just trying to detect whether to use lower case headers or not, I suggest a slightly different approach:

```ruby
LOWER_CASE_HEADERS = Rack::CONTENT_TYPE == "content-type"
```

You could then use that directly or in a method.